docs/api/rest_api/rest_api_reference/input/ez-types.raml

```diff
@@ -2128,401 +2140,1056 @@ View: #TODO outdated example
     type: object
-    description: ''
+    properties:
+        attributeGroups: AttributeGroup[]
 
 AttributeGroupViewInput:
+    description: 'This class represents an attribute group view input.'
     type: object
-    description: ''
 
 AttributeGroupView:
+    description: 'This class represents an attribute group view.'
     type: object
-    description: ''
+    properties:
+        identifier:
+            type: string
+        AttributeGroupList:
+            type: AttributeGroupList
 
 Attribute:
+    description: 'This class represents an attribute.'
     type: object
-    description: ''
+    properties:
+        name:
+            type: string
+        identifier:
+             type: string
+        description:
+             type: string
+        type:
+            type: string
+        type_identifier:
+            type: string
+        value:
+            type: boolean
+        Group:
+            type: AttributeGroup
+        position:
+            type: integer
+        options:
+            type: array
+
 
 AttributeCreate:
+    description: 'This class is used to create an attribute.'
     type: object
-    description: ''
+    properties:
+        identifier:
+             type: string
+        type:
+            type: string
+        group:
+            type: string
+        names:
+            type: array
+        descriptions:
+             type: array
+        positions:
+            type: integer
+        options:
+             type: array
 
 AttributeUpdate:
+    description: 'This class is used to update an attribute.'
     type: object
-    description: ''
+    properties:
+        identifier:
+             type: string
+        names:
+            type: array
+        descriptions:
+             type: array
+        positions:
+            type: integer
+        options:
+             type: array
 
 AttributeList:
+    description: 'This class represents an attribute list.'
     type: object
-    description: ''
+    properties:
+        attributes: Attribute[]
 
 AttributeViewInput:
+    description: 'This class represents an attribute view input.'
     type: object
-    description: ''
+    properties:
+        identifier:
+            type: string
+        AttributeQuery:
+            type: object
 
 AttributeView:
+    description: 'This class represents an attribute view.'
     type: object
-    description: ''
+    properties:
+        identifier:
+            type: string
+        AttributeQuery:
+            type: object
+        Result:
+            type: object
 
 AttributeType:
+    description: 'This class represents an attribute type.'
     type: object
-    description: ''
+    properties:
+        identifier:
+             type: string
+        name:
+            type: string
 
 AttributeTypeList:
+    description: 'This class represents an attribute type list.'
     type: object
-    description: ''
+    properties:
+        attributeTypes
```

Comment:
`Mapping values are not allowed in multi-line blocks at line 2747 (near "attributeTypes").`

Missing colon:

```suggestion
        attributeTypes:
```